hwdb/70-libratbag-mouse.hwdb

```diff
@@ -69,31 +69,39 @@ mouse:usb:v046dpc041:name:*:
  RATBAG_HIDPP10_DPI_LIST=400;800;1600;2000
 
 # G5 2007
+# http://support.logitech.com/en_us/product/g5-laser-mouse-product/specs
 mouse:usb:v046dpc049:name:*:
  RATBAG_DRIVER=hidpp10
  RATBAG_HIDPP10_DPI_LIST=400;800;1600;2000
```

Comment:
is there something missing? If not, the commit msg should point out that those two are unknown, otherwise it looks like a copy/paste error.